stdlib/mem.jou

```diff
@@ -1,13 +1,49 @@
 # Memory management
 
+# TODO: repalce the long with size_t
+# Beacause of many function prototype use size_t:
+# For example:
+#     void * memmove (void * destination, const void * source, size_t num)
+
 # Heap allocations
+
 # TODO: write a tutorial about using these and add a link
+
+# Allocates a block of bytes of memory, returning a pointer to the beginning of the block.
```

Comment:
To be honest I don't like the comments of `malloc()`, `realloc()` and `free()`. I think all Jou users either know about these functions already (because they have experience in C), or need a full tutorial (not just a few comments) for using them.

Reply:
I am not good at writing tutorial(lol)